pkg/policy/distillery_test.go

```diff
@@ -1663,72 +1669,83 @@ func Test_EnsureDeniesPrecedeAllows(t *testing.T) {
 	identity := identity.NewIdentityFromLabelArray(identity.NumericIdentity(identityFoo), labelsFoo)
 
 	tests := []struct {
-		test   string
-		rules  api.Rules
-		result MapState
+		test     string
+		rules    api.Rules
+		expected MapState
 	}{
 		{"deny_world_no_labels", api.Rules{ruleL3DenyWorld, ruleL3AllowWorldIP}, newMapState(map[Key]MapStateEntry{
-			mapKeyL3WorldIngress: mapEntryDeny,
-			mapKeyL3WorldEgress:  mapEntryDeny,
+			mapKeyL3WorldIngress:  mapEntryDeny,
+			mapKeyL3WorldEgress:   mapEntryDeny,
+			mapKeyL3SubnetIngress: mapEntryDeny,
+			mapKeyL3SubnetEgress:  mapEntryDeny,
 		})}, {"deny_world_with_labels", api.Rules{ruleL3DenyWorldWithLabels, ruleL3AllowWorldIP}, newMapState(map[Key]MapStateEntry{
-			mapKeyL3WorldIngress: mapEntryWorldDenyWithLabels,
-			mapKeyL3WorldEgress:  mapEntryWorldDenyWithLabels,
-		})}, {"deny_one_ip_with_a_larger_subnet", api.Rules{ruleL3DenySubnet, ruleL3AllowWorldIP}, newMapState(map[Key]MapStateEntry{
+			mapKeyL3WorldIngress:  mapEntryWorldDenyWithLabels,
+			mapKeyL3WorldEgress:   mapEntryWorldDenyWithLabels,
 			mapKeyL3SubnetIngress: mapEntryDeny,
 			mapKeyL3SubnetEgress:  mapEntryDeny,
+		})}, {"deny_one_ip_with_a_larger_subnet", api.Rules{ruleL3DenySubnet, ruleL3AllowWorldIP}, newMapState(map[Key]MapStateEntry{
+			mapKeyL3SubnetIngress:        mapEntryDeny,
```

Comment:
same question here -- why are there new mapstate entries here?

Reply:
I'm not sure what change caused this, but I'm confident it was not the port range changes.